File: limit2.py
from   ROOT    import *
from   iminuit import Minuit
import numpy   as     np
from   math    import *
import sys


gStyle.SetOptStat(0)
h      = TH1D("h","",40,0,40)

nbkg  = 44.7

nobs  = 0.0
for line in open("dati_highstat.dat"):
    mass = float(line)
    #calcolare nobs
    
    h.Fill(mass)

h.Draw()
gApplication.Run(True)

#calcolare UL @ 95%CL
prob = 0

File: limit2.cpp
#include <iostream>
#include <fstream>
#include <string>
#include <TApplication.h>
#include <TH1D.h>
#include <TMath.h>

using namespace std;

int main(){

  double nbkg = 44.7; 

  TApplication app("app",NULL,NULL);

  TH1D *h = new TH1D("h","",40,0,40);

  ifstream file("dati_highstat.dat");
  
  double mass;
  while (ifile >> mass){
    //calcolare nobs
    h->Fill(mass);
  }

  h->Draw();

  //calcolare UL at 95% CL
  double prob = 0

  
  app.Run(true);


}
